managed-services/webhosting/concepts.mdx

```diff
@@ -19,15 +19,34 @@ An email alias is an additional email address for an email account, with which a
 
 </Concept>
 
+<Contept opened>
+
+## cPanel
+
+cPanel is a web hosting control panel that simplifies the management of websites. It provides a user-friendly graphical interface, allowing users to perform tasks such as creating email accounts, managing files, and installing applications. cPanel is widely used by website owners to streamline the administration of their web hosting services.
+
+</Concept>
+
 <Concept opened>
 
+## Dedicated IP
+
+Dedicated IP is a paid option that provides the capability for each hosting to possess a unique IP, separate from the shared default IP.
+
+Opting for a dedicated IP is ideal if you intend to send emails and safeguard your IP reputation. This ensures that you will not be affected by the negative reputation other clients sharing the same IP may have.
+Upon activation, we automatically assign your hosting a dedicated IP after verifying its absence from any blacklists. Deactivating the option triggers a similar check. If the IP is clean, it will automatically be removed from your account. Otherwise, [contact our support](https://console.scaleway.com/support/tickets) to address any issues.
```

Comment:
```suggestion
Upon activation, we automatically assign your hosting a dedicated IP after verifying it is not blacklisted. Deactivating the option triggers a similar check. If the IP is clean, it will automatically be removed from your account. Otherwise, [contact our support team](https://console.scaleway.com/support/tickets) to address any issues.
```